apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -34,4 +43,1044 @@ function renderSectionProjects(sectionId) {
       element);
   });
 }
-window.renderSectionProjects = renderSectionProjects;
+
+//  Everything below was copied wholesale from index.haml, where we had no linting.
+// TODO (bjvanminnen): Fix remaining lint errors and re-enable rules.
+/* eslint-disable eqeqeq, no-unused-vars */
```

Comment:
I pushed a change to disable the minimum set of rules required for this code to pass eslint.